packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/optimize-ciemss-operation.ts

```diff
@@ -1,5 +1,9 @@
 import { Operation, WorkflowOperationTypes, BaseState } from '@/types/workflow';
-import { Intervention as SimulationIntervention } from '@/types/Types';
+import {
+	Intervention,
+	Intervention as SimulationIntervention,
```

Comment:
Why import this twice with two names?

Reply:
That's weird I never added that line, removing it